### PR TITLE
feat(theme-reset): Reset current theme

### PR DESCRIPTION
**Pull request by Akiletour**
Useful during the theme creation phase.
Allows you to reset the theme (position, modules, ...) without going through the administration interface

If no theme name is passed as an argument. The command will automatically use the active theme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ php bin/console pr:mo install fop_console
 * `fop:maintenance` get status or change maintenance mode, list or add maintenance ip address
 * `fop:generate:htaccess` Generate the .htaccess file
 * `fop:generate:robots`   Generate the robots.txt file
+* `fop:theme-reseet` Reset current (or selected) theme
```
Comment (jf-viguier) [inline README.md:35]: remove one e

Comment (nenes25) [inline README.md:35]: fop:theme-**reset**

File: src/Commands/ThemeResetLayout.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+        $theme = $input->getArgument('theme');
+        $container = $this->getContainer();
+
+        if (!$theme) {
+            $io->text('(?) No theme selected, trying to select current theme...');
```
Comment (jf-viguier) [inline src/Commands/ThemeResetLayout.php:35]: No need for this if only one shop or only one active theme (to be defined)

File: src/Commands/ThemeResetLayout.php [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        try {
+            $themeManager->enable($theme, 1);
+            $themeManager->disable($theme);
+        } catch (\Exception $e) {
+            $io->error(sprintf('The selected theme "%s" is invalid', $theme));
+
+            return 1;
+        }
+
+        $io->success(sprintf('Theme "%s" resetted with success.', $theme));
```
Comment (jf-viguier) [inline src/Commands/ThemeResetLayout.php:65]: remove one t I guess

File: src/Commands/ThemeResetLayout.php [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+        $theme = $input->getArgument('theme');
+        $container = $this->getContainer();
+
+        if (!$theme) {
```
Comment (zalexki) [inline src/Commands/ThemeResetLayout.php:34]:
```suggestion
        if (null !== $theme) {
```

File: src/Commands/ThemeResetLayout.php (hunk 2 above)
Comment (zalexki) [inline src/Commands/ThemeResetLayout.php:35]: maybe ask a confirmation before actually do the reset in that case of select auto default theme ?

File: src/Commands/ThemeResetLayout.php [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+        $themeManager = $this->getContainer()->get('prestashop.core.addon.theme.theme_manager');
+
+        try {
+            $themeManager->enable($theme, 1);
+            $themeManager->disable($theme);
+        } catch (\Exception $e) {
+            $io->error(sprintf('The selected theme "%s" is invalid', $theme));
+
+            return 1;
```
Comment (zalexki) [inline src/Commands/ThemeResetLayout.php:62]:
```suggestion
            return Command::FAILURE;
```

File: src/Commands/ThemeResetLayout.php (hunk 2 above)
Comment (jf-viguier) [reply src/Commands/ThemeResetLayout.php]: @zalexki I don't think so : in this case there is only one theme, no need to add this IMO.

Comment (zalexki) [reply src/Commands/ThemeResetLayout.php]: ye ok, that's a user experience you have to choose of course :)

**Review by jf-viguier — APPROVED** [review]
I've tested your PR, It works well, thanks

**Review by nenes25 — APPROVED** [review]

**Pull request merged**